app/assets/stylesheets/govuk_publishing_components/components/_action-link.scss

```diff
@@ -150,6 +150,24 @@
   }
 }
 
+.gem-c-action-link--dark-large-icon {
+  max-width: none;
+
```

Comment:
I'm not quite sure why `max-width: none;` is needed @MartinJJones - any idea?

Reply:
It appears the component sets a `max-width` of `410px` for screen sizes up to `$tablet` (641px).

`gem-c-action-link` - https://github.com/alphagov/govuk_publishing_components/blob/main/app/assets/stylesheets/govuk_publishing_components/components/_action-link.scss#L19

When testing locally, this would not make a difference given the current length of text, but if more text was added in the future it would have a slight impact, for example:

### Screen width 640px

#### max-width set to none
![Screenshot 2023-09-15 at 10 44 43](https://github.com/alphagov/govuk_publishing_components/assets/28779939/ea2cf4dc-a67a-4fba-a715-eb7ae02d2766)

#### max-width left as default - 410px
![Screenshot 2023-09-15 at 10 44 21](https://github.com/alphagov/govuk_publishing_components/assets/28779939/4523849c-dfcb-4a4a-a3d0-518932a78771)

So I think this adding or removing it would come down to preference of how it should appear on tablet when we have a lot of text.

Reply:
Thanks for the comment Jon and for digging into it Martin! I've removed as from your investigation, it only impacts longer text than we're currently proposing so not value in keeping `max-width`